statie/tests/Posts/Year2017/ListeningNetteComponents/ListeningNetteComponentsTest.php

```diff
@@ -61,6 +65,8 @@ public function testAddToBasketFirstProductRequest(): void
 
     public function testAddToBasketSecondProductRequest(): void
     {
+        $this->markTestSkipped('Broken since Nette 3.0');
+
         $request = new Request(self::PRESENTER_NAME, 'GET', ['do' => 'addToBasket-2-add']);
         $presenter = $this->createPresenter();
```

Comment:
@TomasPilar Hi Tom, 2 years ago you wrote tests for the components with listeners to your article on Pehapkari.

Since Nette 3.0 there is BC break change, that breaks the url.

`['do' => 'addToBasket-2-add']` doesn't work anymore and it returns `RedirectResponse`

Any ideas to fix this?

Thank you